src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
```

Comment:
It would be great if you will create packages and put figures in 'figure' package, suppliers to 'supplier'